### ISSUE TITLE
guide to pytorch implement

### ISSUE THREAD

**Issue by harlem867** [xuannianz/keras-GaussianYOLOv3#7]
@xuannianz Hello, i am trying to implement guassian yolov3 in pytorch,
and i am using your guassian loss function.
During training, the guassian loss of x,y,w,h can reduce as normal, but in the test, all the test mAPs are 0.   So i checked the training process, and have found that the mean of x,y,w,h didn't approach to the true value of GT while the sigmas can be rightly reduced to zero.
Can you give some advice? thanks a lot :）

**Comment by xuannianz** [comment]
Hi, how do you find the means do not approach to the gt values? Did the losses about x,y,w,h decrease?

**Comment by harlem867** [comment]
1）yes，the gaussian loss of all the x,y,w,h can be rightly reduced. But while testing, the mAP is 0.
2）So I checked the training process, caculate and print the MSE between x and mean_x(the true value) and the MSE between sigma and 0, and find the means do not approach to the gt values, so the predicted result is not right.
